recipes/functorch/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "functorch" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/functorch/archive/v{{ version }}.tar.gz
+  sha256: 3b297f4acf1d61fd95b86bd2ec1805bc9c236bb0fb341a42c2300a4b3d497fbd
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

```suggestion
  skip: true  # [win]
  skip: true  # [cuda_compiler_version != None]
```
Doesn't seem like this package needs the cuda compiler, but apparently our CI things any mention of "cuda" should activate the cuda builds.